vignettes/REFERENCES.bib

```diff
@@ -1,12 +1,15 @@
-
 @article{zhuAutocorrelatedBayesian2023,
-	title = {The {Autocorrelated} {Bayesian} {Sampler}: {A} rational process for probability judgments, estimates, confidence intervals, choices, confidence judgments, and response times},
-	doi = {10.1037/rev0000427},
-	journal = {Psychological Review},
-	author = {Zhu, Jian-Qiao and Sundh, Joakim and Spicer, Jake and Chater, Nick and Sanborn, Adam N.},
-	year = {2023},
-	keywords = {ObsCite},
+  title = {The Autocorrelated {{Bayesian}} Sampler: {{A}} Rational Process for Probability Judgments, Estimates, Confidence Intervals, Choices, Confidence Judgments, and Response Times.},
+  shorttitle = {The Autocorrelated {{Bayesian}} Sampler},
+  author = {Zhu, Jian-Qiao and Sundh, Joakim and Spicer, Jake and Chater, Nick and Sanborn, Adam N.},
+  year = {2023},
```

Comment:
I think Psych Review has just updated this article with the number of volume and pages. (https://psycnet.apa.org/fulltext/2023-78840-001.html)